src/Blocks/components/checkboxes/checkboxes.php

```diff
@@ -10,13 +10,33 @@
 
 $manifest = Components::getManifest(__DIR__);
 
+$checkboxesId = Components::checkAttr('checkboxesId', $attributes, $manifest);
 $checkboxesContent = Components::checkAttr('checkboxesContent', $attributes, $manifest);
+$checkboxesName = Components::checkAttr('checkboxesName', $attributes, $manifest);
+
+// Add internal counter name key.
+$checkboxesContent = (string) preg_replace_callback('/name=""/', function () use ($checkboxesName) {
```

Comment:
You can use just regular `preg_replace` here, no? Since you have just one parameter to replace.